src/django/api/moderation_event_actions/approval/event_approval_template.py

```
@@ -9,6 +9,7 @@
 from api.constants import (
     LOCATION_CONTRIBUTION_APPROVAL_LOG_PREFIX,
     ProcessingAction,
+    APIV1MatchTypes,
```

Ruff (0.8.2)

`api.constants.APIV1MatchTypes` imported but unused

Remove unused import: `api.constants.APIV1MatchTypes`

(F401)